examples/popmotion/index.html

```diff
@@ -16,7 +16,7 @@
 </head>
 <body>
   <div id="ball"></div>
-  <script src="https://unpkg.com/popmotion@8.1.0/dist/popmotion.global.min.js"></script>
+  <script src="https://unpkg.com/popmotion@8.7.5/dist/popmotion.global.min.js"></script>
```

Comment:
Hey @Tomekmularczyk, we'd actually like to move away from using CDN urls for these resources, by saving a copy of the file and committing it to the repo. I should have been a bit more clear about this in [the first bullet point in the issue description](https://github.com/sparkbox/bouncy-ball/issues/87#issue-708279707).

Could you also make that update?

Reply:
ah, missunderstanding then, I'll change it